src/test/scala/nl/knaw/dans/easy/umd/CommandSpec.scala

```diff
@@ -157,21 +158,72 @@ class CommandSpec extends FlatSpec
 
   it should "reject CSV with other encoding than UTF-8" in {
 
-    val file = new File("src/test/resources/mac-encoded.txt")
+    val file = new File("src/test/resources/KOI8-R.txt")
     implicit val ps = Parameters(test = true,fedoraCredentials = null, input = file)
     Command.testFriendlyRun.failed.get.getMessage shouldBe
-      "requirement failed: encoding of src/test/resources/mac-encoded.txt must be UTF-8 but is KOI8-R"
+      "requirement failed: encoding of src/test/resources/KOI8-R.txt must be one of [UTF-8, ISO-8859-1] but is KOI8-R"
   }
 
   "textReader" should "not try to recover from not expected encoding" in {
 
     // saved "ÈÀŒØ" with Mac's text editor with "Western (Mac OS Roman)"
-    val file = new File("src/test/resources/mac-encoded.txt")
+    val file = new File("src/test/resources/KOI8-R.txt")
 
     val reader = Command.textReader(file).get
-    reader.getCharset.toString shouldBe "KOI8-R"
-    reader.readLine() shouldBe "Икн╞"
+    reader.getCharset shouldBeRejectedCharset "KOI8-R"
+    reader.readLine shouldBe "Икн╞"
     FileUtils.readFileToString(file,"KOI8-R") shouldBe "Икн╞"
     FileUtils.readFileToString(file,"UTF-8") shouldBe "��ί"
   }
+
+  it should "accept plain ascii" in {
+
+    val file = new File("src/test/resources/ISO-8859-1.txt")
+
+    val reader = Command.textReader(file).get
+    reader.getCharset shouldBeAcceptedCharset "ISO-8859-1"
+    reader.readLine shouldBe "created with intelij on a mac"
+  }
+
+  it should "accept vi file with plain ascii" in {
+
+    val file = new File("src/test/resources/ascii.txt")
+
+    val reader = Command.textReader(file).get
+    reader.getCharset shouldBeAcceptedCharset "ISO-8859-1"
+    reader.readLine shouldBe "created with vi"
+  }
+
+  it should "accept vi file with Dutch accent" in {
+
+    val file = new File("src/test/resources/vi-accent.txt")
+
+    val reader = Command.textReader(file).get
+    reader.getCharset shouldBeAcceptedCharset "ISO-8859-1"
+    reader.readLine shouldBe "reëel accent"
+  }
+
+  it should "accept vi file with euro sign" in {
+
+    val file = new File("src/test/resources/vi-euro.txt")
+
+    // https://en.wikipedia.org/wiki/ISO/IEC_8859
+    // would rather expect 8859-15
+    val reader = Command.textReader(file).get
+    reader.getCharset shouldBeAcceptedCharset "ISO-8859-1"
+    reader.readLine shouldBe "euro: €"
+  }
+
+  implicit class CS (actual: Charset) {
```

Comment:
I like you create implicit classes to extend the API! However, it is good practise to add `extends AnyVal`. See http://docs.scala-lang.org/overviews/core/value-classes.html